src/test/java/api/support/builders/FeeFineOwnerBuilder.java

```diff
@@ -0,0 +1,80 @@
+package api.support.builders;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import java.util.List;
+import java.util.UUID;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineOwnerBuilder extends JsonBuilder implements Builder {
+  private UUID id;
+  private String owner;
+  private String desc;
+  private List<JsonObject> servicePointOwner;
+  private String defaultChargeNoticeId;
+  private String defaultActionNoticeId;
+  private JsonObject metadata;
```

Comment:
Similarly, why does this builder need to define the change metadata?

Reply:
Removed metadata field.